src/components/IOUConfirmationList.js

```diff
@@ -232,6 +279,32 @@ class IOUConfirmationList extends Component {
         return iouAmount !== sumAmount ? (amountPerPerson + difference) : amountPerPerson;
     }
 
+    /**
+    * Toggle selected option's selected prop.
+    * @param {Object} option
+    */
+    toggleOption(option) {
+        const isSelf = _.every(this.state.participants, selectedOption => (
```

Comment:
Not following what `isSelf` is referring to here.

Reply:
We don't wanna toggle option for self.
The image below highlights self.

<img width="261" alt="Untitled" src="https://user-images.githubusercontent.com/29673073/125367004-fd99d100-e37f-11eb-93b6-41e8fbb31aa8.png">

Please let me know if there's a better name for it.

Reply:
So "self" is the currently logged in user? Or someone else?

Reply:
If that's right, it's unnecessary to loop over the participants to check this. This user should already be in this list. So maybe we can instead return early if `session.email === option.login` and subscribe to session email? Let me know if that makes sense or if I have something incorrect.

Reply:
Yes, it's the currently loggin in user.


> If that's right, it's unnecessary to loop over the participants to check this. This user should already be in this list. So maybe we can instead return early if `session.email === option.login` and subscribe to session email? Let me know if that makes sense or if I have something incorrect.

Sounds great!